src/compat/tracker.jl

```diff
@@ -225,102 +225,6 @@ end
     end
 end
 
-for header in [
```

Comment:
All this ain't necessary anymore? Sweet-sauce!

Reply:
Yep, everything seems to work :slightly_smiling_face: I even checked the example from the README, it works and the types (TrackedArray etc.) are the same.

Reply:
Dope stuff:)